lib/ClangImporter/ImportDecl.cpp

```diff
@@ -5800,7 +5805,7 @@ namespace {
       if (dc == Impl.importDeclContextOf(decl, decl->getDeclContext())) {
         auto known = Impl.ImportedDecls.find({decl->getCanonicalDecl(),
                                               getVersion()});
-        if (known != Impl.ImportedDecls.end())
+        if (known != Impl.ImportedDecls.end() && known->second != nullptr)
```

Comment:
Same as above. Why can't we return nullptr here?

Reply:
I actually only added this one back because Thomas had it in the previous patch. Everything passes fine with this line returning nullptr.